pkgs/development/python-modules/stytra/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib, pkgs, buildPythonPackage, fetchPypi, isPy3k, callPackage
+, opencv3
+, pyqt5
+, pyqtgraph
+, numpy
+, scipy
+, numba
+, pandas
+, tables
+, git
+, ffmpeg
+, scikitimage
+, matplotlib
+, qdarkstyle
+, GitPython
+, anytree
+, pims
+, imageio
+, imageio-ffmpeg
+, av
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "stytra";
+  version = "0.8.26";
+  format = "wheel";
+  disabled = !isPy3k;
+
+  arrayqueues = callPackage ./arrayqueues.nix {};
+  colorspacious = callPackage ./colorspacious.nix {};
+  qimage2ndarray = callPackage ./qimage2ndarray.nix {};
+  flammkuchen = callPackage ./flammkuchen.nix {};
+  lightparam = callPackage ./lightparam.nix {};
+
+  src = fetchPypi {
+    inherit pname version;
+    format = "wheel";
```

Comment:
you should prefer source over wheel... always
```suggestion
```

Reply:
@FRidh's [post](https://github.com/NixOS/nixpkgs/pull/64047#issuecomment-522314445) made me think that building wheel is preferred?

Reply:
there's two things going on.... using a wheel from pypi, and building a wheel locally.

We prefer to receive source, and then as part of the buildPythonPackage workfow, build a wheel then install the wheel. This generally allows us to get tests, while only installing the files that package maintainer meant to distribute with a wheel.

We should prefer not to receive a wheel from pypi unless source isn't available from pypi or the upstream repo

Reply:
ah, thanks for the clarification!